homeassistant/components/starlink/sensor.py

```diff
@@ -110,4 +94,24 @@ def native_value(self) -> StateType | datetime:
         entity_category=EntityCategory.DIAGNOSTIC,
         value_fn=lambda data: now() - timedelta(seconds=data.status["uptime"]),
     ),
+    StarlinkSensorEntityDescription(
+        key="azimuth",
+        name="Azimuth",
+        icon="mdi:compass",
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        native_unit_of_measurement=DEGREE,
+        entity_registry_enabled_default=False,
+        value_fn=lambda data: round(data.status["direction_azimuth"]),
+    ),
+    StarlinkSensorEntityDescription(
+        key="elevation",
+        name="Elevation",
+        icon="mdi:compass",
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        native_unit_of_measurement=DEGREE,
+        entity_registry_enabled_default=False,
+        value_fn=lambda data: round(data.status["direction_elevation"]),
+    ),
```

Comment:
I'm assuming these change frequently as it moves to track the sat?

Reply:
Yep, if they're not changing then something is wrong 😆

I think the value is rounded, so users might not see an update depending how fast the tracked satellite is moving